docs/proposals/scheduling/policy-preemption/README.md

```diff
@@ -0,0 +1,256 @@
+        - member1
+```
+
+This proposal proposes a strategy which allows policies to support preemption by priority at runtime.
+Even if workloads have been propagated by a policy, they can be preempted by a high priority of policy.
+
+## Motivation
+
+Cluster administrators usually cannot foresee future expansion scenarios when configuring policies.
+They will usually start with a broad policy to set the base strategy. When an application requires special configuration,
+the administrator wants to provide a persionalized policy to take over the application.
+At this time, it hopes that the high-priority policy can preempt the low-priority policy.
+
+### Goals
+
+- Extend the API of PropagationPolicy/ClusterPropagationPolicy to allow high-priority policies to preempt low-priority policies.
+- Propose the implementation ideas for involved components, including `karmada-controller-manager`.
+- Propose a solution that is compatible with previous versions. The preemption capability will be disabled by default, 
+upgrading the system from the previous version to the new one will not result in any changes in behavior and will not require any additional adaptation work.
+
+### Non-Goals
+
+## Proposal
+
+### User Stories (Optional)
+
+#### As a cluster administrator, I hope that the high-priority propagationPolicy can preempt the low-priority propagationPolicy.
+
+As a cluster administrator, I am responsible for unified management of multi-cluster scheduling policies. 
+Workloads are responsible for each business team. For the current businesses under the same namespace, I deploy a generic policy to match all.
+At some point, I want to change the scheduling strategy of one of the businesses, so I create a higher-priority policy.
+I hope that the high-priority policy can preempt the low-priority policy.
+
+#### As a cluster administrator, I hope that the propagationPolicy can match resources that have been matched by clusterPropagationPolicy under the same namespace.
+
+As a cluster administrator, I am responsible for unified management of multi-cluster scheduling policies.
+Workloads are responsible for each business team. For the current business, I deploy a cpp to match all namespace-scoped resources.
+At some point, I want to change the scheduling strategy of one of the businesses under one of namespaces, so I create a pp to match it only.
+I hope that the propagationPolicy can match it that have been matched by clusterPropagationPolicy.
+
+### Risks and Mitigations
+
+This proposal maintains the backward compatibility, the system built with previous
+versions of Karmada can be seamlessly migrated to the new version.
+The previous configurations(yamls) could be applied to the new version of Karmada and
+without any behavior change.
+
+### Notes/Constraints/Caveats (Optional)
+
+The priority based on preemption do not include implicit priority when `.spec.priority` of policies are the same.
+
+For example, the following two policies do not preempt.
```

Comment:
Do you mean that `propagation-namematch` won't preempt `propagation-labelselectormatch` even if preemption is enabled?

Reply:
Yeah.